unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -349,12 +359,18 @@ private async UniTask ProcessIncomingFriendshipEvent(FriendshipEventResponse fri
                     cancellationToken.ThrowIfCancellationRequested();
 
                     OnFriendRequestAccepted?.Invoke(friendshipEvent.Accept.User.Address);
+
+                    // Notify kernel that there's a new friend
```

Comment:
perhaps you can add TODO comments to delete the calls once the friendships kernel are completely deprecated